src/audio/pipeline.c

```diff
@@ -1090,6 +1106,17 @@ static int pipeline_xrun_recover(struct pipeline *p)
 		return ret;
 	}
 
+	/* for playback copy it here, because scheduling won't work
+	 * on this interrupt level
+	 */
```

Comment:
So you mean you kept the schedule idle copy in the pipeline trigger start as it will never work there?
Thus this is some workaround for now?

Reply:
Exactly. Also my comment is a little bit misleading. To be precised, schedule won't work, because scheduler doesn't allow to add to the list tasks, which are currently running.